include/antler/project/yaml.hpp

```diff
@@ -0,0 +1,110 @@
+#pragma once
+
+/// @copyright See `LICENSE` in the root directory of this project.
+
+
+#include <string>
+#include <string_view>
+#include <vector>
+#include <iostream>
+#include <utility> // std::pair
+#include <fstream>
+
+#include "../system/utils.hpp"
+
+#include "yaml-cpp/yaml.h"
+
+namespace antler::project {
+   struct yaml {
+      using node_t = YAML::Node;
+
+      inline static node_t load(const system::fs::path& path) { return YAML::LoadFile(path.string()); }
+
+      inline static void write(system::fs::path path, const node_t& node) {
+         std::ofstream out(path);
+         out << node;
+         out.close();
```

Comment:
If we aren't checking for or logging errors, why are we manually `closing()`ing instead of the destructor?